gradle/root/release.gradle

```diff
@@ -23,23 +23,22 @@ ext {
     git_genericEmail = "<mockito.release.tools@gmail.com>"
     git_releasableBranchRegex = "master|release/.+"  // matches 'master', 'release/2.x', 'release/3.x', etc.
 
-    bintray_repo = 'maven'
-    bintray_notableRepo = 'maven-notable'
-
     pom_developers = ['szczepiq:Szczepan Faber', 'bric3:Brice Dutheil', 'raphw:Rafael Winterhalter', 'TimvdLippe:Tim van der Lippe']
     pom_contributors = []
 }
 
 allprojects {
-    plugins.withId("org.mockito.release-tools.bintray") {
+    plugins.withId("org.mockito.mockito-release-tools.bintray") {
         bintray {
             pkg {
+                repo = 'maven'
                 user = 'szczepiq'
```

Comment:
It's not a bad idea. Problem is that it would be more work (user to maintain, store password safely, etc.). We're authenticating safely using bintray api key that is bound to a user. If I'm not around you can always replace this user with one of the other maintainers and update Travis env variable BINTRAY_API_KEY. For now, I'll leave myself here. However, this deserves some documentation somewhere.